app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def app_user_check(_post, _user, current_user)
```

Comment:
This method signature doesn't need parameters at all, this is because helpers are in the context of views, and the `current_user` method is available there (because it is available too in the controller's context.
```suggestion
  def app_user_check
```